app/boards/shields/tidbit/README.md

```diff
@@ -0,0 +1,50 @@
+# Building ZMK for the Tidbit
+
+Some general notes/commands for building standard Tidbit layouts from the assembly documentation.
+
+## Standard "Non Dense" Build
+
+```
+
+west build -p --board nice_nano -d build/tidbit/default -- -DSHIELD=tidbit
+
+```
+
+## Dense "19 keys" Build
+
+```
+
+west build -p --board nice_nano -d build/tidbit/19_key -- -DSHIELD=tidbit_19key
+
+```
+
+## LED Notes
+
+If you built your tidbit without the LEDs *and* are using a Nice!Nano board, you'll need to add the following at the *end* of your local tidbit config.
+
+```
+
+CONFIG_ZMK_RGB_UNDERGLOW=n
+CONFIG_WS2812_STRIP=n
+
+```
+
+## Encoder Notes
+
+If you built your tidbit without encoders, you'll need to add the following at the *end* of your local tidbit config.
+
+```
+
+CONFIG_EC11=n
+CONFIG_EC11_TRIGGER_GLOBAL_THREAD=n
+
+```
+
+## OLED Builds
+
+If using an OLED screen you'll need to include the following at the *end* of your local tidbit config.
+
+```
+CONFIG_ZMK_DISPLAY=y
+
+```
```

Comment:
I like this a lot more although I'm wondering if adding these to the end is easier than saying to set these config props to x value. Seems a bit weird to have two sets of the same config options in the same file.

Reply:
These are just config options that override the core zmk repo config options. They should be applied to the zmk-config dir / personal tweaks zones. I'd rather have them in the config per previous discussions than as build parms.

Thoughts?

Reply:
Yes, I'm totally on board with these config options. What I'm confused about is copying these to the conf file when these config options already exist in the .conf file that's copied in using zmk-config and is already available if building from source.

Reply:
Ohhhh, I'll swap the note to change them if present or add them to the end of the config if they aren't already present.